maven-settings/src/main/mdo/settings.mdo

```diff
@@ -581,7 +581,18 @@
             ]]>
           </description>
         </field>
-      </fields>
+        <field>
+          <name>ids</name>
+          <version>1.3.0+</version>
+          <description>
+            List of additional ids for server.
```

Comment:
What happens if the listed ids have separate entries? Are they merged? Which ones takes precedence?

Reply:
Good one.